src/App.test.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
-
+//testing
```

Comment:
please remove comments that no longer need to be in the code

Reply:
good catch Janet